doc/API.md

```diff
@@ -737,6 +787,21 @@ lists the status of all files.
 }
 ```
 
+#### /renter/prices [GET]
+
+lists the estimated prices of performing various storage and data operations.
+
+###### JSON Response [(with comments)](/doc/api/Renter.md#json-response-4)
+```javascript
+{
+  "downloadterabyte":      "1234", // hastings
+  "formcontracts":         "1234", // hastings
+  "storageterabytemonth":  "1234", // hastings
+  "uploadterabyte":        "1234", // hastings
```

Comment:
I agree that this is a controversial choice. I stand behind it, but not very strongly.